var/spack/repos/builtin/packages/nccmp/package.py

```diff
@@ -0,0 +1,30 @@
+from spack import *
+import os
+
+class Nccmp(Package):
+    """Compare NetCDF Files"""
+    homepage = "http://nccmp.sourceforge.net/"
+    url      = "http://downloads.sourceforge.net/project/nccmp/nccmp-1.8.2.0.tar.gz"
+
+    version('1.8.2.0', '81e6286d4413825aec4327e61a28a580')
+
+    depends_on('netcdf')
+
+    def install(self, spec, prefix):
+        # Configure says: F90 and F90FLAGS are replaced by FC and
+        # FCFLAGS respectively in this configure, please unset
+        # F90/F90FLAGS and set FC/FCFLAGS instead and rerun configure
+        # again.
+        os.environ['FC'] = os.environ['F90']
+        del os.environ['F90']
+        try:
+            os.environ['FCFLAGS'] = os.environ['F90FLAGS']
+            del os.environ['F90FLAGS']
+        except KeyError:    # There are no flags
```

Comment:
From [this thread](http://www.gossamer-threads.com/lists/python/python/1015052), apparently the preferred pattern is based on the likelihood of the key existing in the dictionary:

```
> I was looking at the example found here [1] which begins with: 
> 
> [1] http://docs.python.org/py3k/library/imp.html#examples 
> 
> def __import__(name, globals=None, locals=None, fromlist=None): 
> # Fast path: see if the module has already been imported. try: 
> return sys.modules[name] 
> except KeyError: 
> pass 
> 
> I was wondering if the formulation 
> 
> if name in sys.modules: 
> return sys.modules[name] 
> 
> would be equivalent. IOW, is using try/except here only a matter of 
> style or a necessity? 

Mostly style, but not entirely. 

If you expect that most of the time the module will be found, the 
try...except version will be faster. If you expect that most of the time 
the module will not be found, the "if name in" version will be faster. 
```

Not that this is a performance-sensitive code path.